server/routes.js

```diff
@@ -108,12 +108,27 @@ function getDomainSlug(publisherConfig, hostName) {
   return publisherConfig.domain_mapping[hostName] || null;
 }
 
-function withConfigPartial(getClient, logError, publisherConfig = require("./publisher-config")) {
+const isFreshRevisionFn = (revision, assetHash, pbConfigVersion) => revision !== `${assetHash}-${pbConfigVersion}`;
+
+const maxConfigVersionFn = (config = {}, enablePb) => {
```

Comment:
Can't pass maxConfigVersion from app as it's publisher specific, we need it before load-data and couldn't figure out a way to get the respective publisher details on server app start